package.json

```diff
@@ -23,10 +23,10 @@
     "gulp-shell": "^0.5.2",
     "jquery": "~2.2.0",
     "moment": "^2.15.1",
-    "moment-timezone": "~0.5.5",
+    "moment-timezone": "0.5.5",
     "requirejs": "~2.1.22",
     "requirejs-text": "~2.0.12",
-    "sinon": "1.17.3 || >1.17.4",
+    "sinon": "1.17.3 || 1.17.7",
```

Comment:
would just go to `^1.17` for this, the `||` I put here a few months ago because `1.17.4` was a broken release